packages/ui/src/components/multi-select-v2.tsx

```diff
@@ -309,13 +313,19 @@ export const MultiSelect = forwardRef<MultiSelectRef, MultiSelectProps>(
               >
                 {isLoading ? (
                   <SkeletonList />
-                ) : availableOptions.length === 0 ? (
-                  <Command.Item value="-" disabled>
-                    No results found
-                  </Command.Item>
+                ) : availableOptions?.length === 0 ? (
+                  disallowCreation ? (
+                    <Command.Item value="-" disabled>
+                      No results found
+                    </Command.Item>
+                  ) : (
+                    <Command.Item value="-" disabled>
+                      Press enter to create value
```

Comment:
we will need product/copy guidance on this. for now let's change it to just `Press Enter to create`